### PR TITLE
[FEATURE] - MS-35: Sessions redux

### PR DESCRIPTION
**Pull request by BaltazarCortina**
### Description

Implemented redux in Sessions components

### CODE REVIEW

File: src/Components/helpers/index.js [hunk 1]
```diff
@@ -6,9 +6,10 @@ export const capitalize = (str) => {
   return words.join(' ');
 };
 
-export const removeLastChar = (str) => str.substr(0, str.length - 1);
-
-export const fixCompaniesTitle = (str) => str.substr(0, str.length - 3) + 'y';
+export const removeLastChar = (str) => {
+  if (str === 'companies') return 'company';
```
Comment (Sabrucita) [inline src/Components/helpers/index.js:10]: Ln 10 it's like this:   

if (str === 'Companies') return 'Company';

first capital letter

**Review by Sabrucita — APPROVED** [review]
Approved 😊

File: src/Components/helpers/index.js (hunk 1 above)
Comment (BaltazarCortina) [reply src/Components/helpers/index.js:10]: You're right, we're capitalizing it first so the C is going to be uppercase

**Review by EugenioPonzone — APPROVED** [review]

File: src/redux/sessions/thunks.js [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+    dispatch(deleteSessionFetching());
+    dispatch(updateModal('fetching'));
+    fetch(`${url}/sessions/${id}`, {
+      method: 'DELETE'
+    })
+      .then(async (res) => {
+        if (res.status === 200) {
+          const data = await res.json();
+          dispatch(deleteSessionFulfilled(data));
+          return dispatch(updateModal('deleted', data.data));
```
[the remark this answers is not in the document]
Comment (BaltazarCortina) [reply src/redux/sessions/thunks.js:150]: Fixed

**Pull request merged**